setup.py

```diff
@@ -7,6 +7,7 @@
 
 dependencies = [
     "blspy>=0.9",
+    "clvm_rs>=0.1.8"
```

Comment:
I think it's best to make `clvm_rs` an optional dependency. I expect that `clvm_rs` will eventually be completely standalone.